dnd/index.js

```diff
@@ -2,20 +2,29 @@ const express = require('express')
 const app = express()
 const cors = require('cors')
 const bodyParser = require('body-parser')
+const morgan = require('morgan')
 const PORT = 5000
 
 const characters = []
 
+app.use(bodyParser.json())
+
 app.use((req, res, next) => {
-  res.header("Access-Control-Allow-Origin", "*");
-  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
-  next();
+  res.header("Access-Control-Allow-Origin", "*")
+  res.header("Access-Control-Allow-Headers",
+  "Origin, X-Requested-With, Content-Type, Accept", "Authorization")
+  if(req.method === 'OPTIONS') {
+    res.header('Access-Control-Allow-Methods', 'GET, POST, PUT, PATCH, DELETE')
+    return res.status(200).json({})
+  }
```

Comment:
Maybe try to make use of `cors()` instead https://www.npmjs.com/package/cors

Reply:
Last time we tried that it didn't work!

Reply:
Lets figure it out! :100: